script.js

```diff
@@ -329,5 +329,28 @@ formButton.addEventListener('click', (event) => {
     event.preventDefault();
     return;
   }
+
   formButton.submit();
 });
+
+/* Local Storage */
+
+const formName = document.getElementById('name');
+const formMessage = document.getElementById('message');
+
+function saveData() {
+  const contactForm = document.getElementById('contact');
+  const formData = new FormData(contactForm);
+  const formObject = Object.fromEntries(formData.entries());
+  localStorage.setItem('form', JSON.stringify(formObject));
+}
+
+email.addEventListener('keydown', saveData);
+formName.addEventListener('keydown', saveData);
+formMessage.addEventListener('keydown', saveData);
+
+const savedData = JSON.parse(localStorage.getItem('form') ?? '{}');
+
```

Comment:
While you did a good job implementing the project requirements there are still some issues you need to fix inorder for your project to get approved. I noticed you disabled eslint in your code, it is against microverse policy to disable linters because linters help us write clean code so it would be nice if you enabled eslint. I have attached some screen shots to show you were you disabled eslint .

![snip2](https://user-images.githubusercontent.com/79506105/209346077-932ceb63-1017-468e-937d-674e8ae125d1.PNG)

![Snip 1](https://user-images.githubusercontent.com/79506105/209346086-40c77e63-648b-4efe-b785-ad40c64e6be5.PNG)

Please kindly correct this issue.

Reply:
@KayLemba okay thanks for the suggestion. I fixed the issue you highlighted but now I am having a problem submitting it for code review.